docs/sources/http_api/dashboard_versions.md

```diff
@@ -65,6 +67,66 @@ Status Codes:
 - **401** - Unauthorized
 - **404** - Dashboard version not found
 
+## Get all dashboard versions by dashboard UID
+
+Query parameters:
+
+- **limit** - Maximum number of results to return
+- **start** - Version to start from when returning queries
+
+`GET /api/dashboards/uid/:uid/versions`
+
+Gets all existing dashboard versions for the dashboard with the given `uid`.
+
+**Example request for getting all dashboard versions**:
+
+```http
+GET /api/dashboards/uid/QA7wKklGz/versions?limit=2?start=0 HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+```
+
+**Example Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=UTF-8
+Content-Length: 428
+
+[
+  {
+    "id": 2,
+    "dashboardId": 1,
+    "uid": "QA7wKklGz",
```

Comment:
Hi @yangkb09. Thanks for these changes. I'm working on replacing ID by UID, and I found some inconsistencies with the API.
The response when using `api/dashboards/uid/${uid}/versions` is 
<img width="1456" alt="Captura de Pantalla 2022-07-08 a las 17 09 05" src="https://user-images.githubusercontent.com/5699976/178020337-14a06371-85fe-4daf-a129-0a87c8b5da24.png">

It seems the field returned is not `uid`, it's `dashboardUID`. Also, the field comes always as an empty array as you can see in the screenshot. Do we know why?